### ISSUE TITLE
Hi wafflejock

### ISSUE THREAD

**Issue by sonvirgo** [sonvirgo/i2c-rotation#1]
If you found st pls notify me 
Thanks!

**Comment by shusain** [comment]
Yup wafflejock on IRC here will watch the project to see what you're working on and if I can help.

**Comment by sonvirgo** [comment]
Thank you very much. Please accept the invitation!

**Comment by sonvirgo** [comment]
Hi  shusain, although the driver must lay some where the kernel source. Actual desktop system currently using this as I have seen only Gnome. Ubuntu to Unity, Fedora and RedHat to available in rotation.
How about asking people in Gnome community?
